requirements.txt

```diff
@@ -0,0 +1,22 @@
+# for all
+numpy>=1.24.2
+wandb>=0.13.10
+matplotlib>=3.7.0
+scipy>=1.10.0
+pytorch-lightning>=2.0.3
+shapely>=2.0.1
+networkx>=3.0
+imageio>=2.34.0
+numcodecs>=0.12.1
+Cartopy>=0.22.0
+pyproj>=3.4.1
+xarray>=2024.1.1
+tueplots>=0.0.8
+plotly>=5.15.0
+# for dev
```

Comment:
normally you would do a two-stepped approach here where you specify the two requirements in two files. See here for example:
https://github.com/MeteoSwiss-APN/nwp_log_mining/blob/main/requirements/requirements_build.txt
https://github.com/MeteoSwiss-APN/nwp_log_mining/blob/main/requirements/requirements_run.txt

Reply:
This is directly defined by upstream. If you have strong feelings about it I can create a PR? otherwise I would suggest to use the conda within MeteoSwiss as it is easier and consistent with our common practices.